project/Dependencies.scala

```diff
@@ -0,0 +1,359 @@
+    "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % JacksonVersion,
+    "com.typesafe.akka" %% "akka-testkit" % AkkaVersion % "test",
+    scalaTest % Test,
+    "com.novocode" % "junit-interface" % "0.11" % "test"
+  )
+
+  val `play-json` = libraryDependencies ++= Seq(
+    "com.typesafe.play" %% "play-json" % PlayVersion,
+    "com.typesafe.akka" %% "akka-actor" % AkkaVersion,
+    "com.typesafe.akka" %% "akka-testkit" % AkkaVersion,
+    scalaTest % Test
+  )
+
+  val `api-tools` = libraryDependencies ++= Seq(
+    "com.typesafe.play" %% "play" % PlayVersion,
+    scalaTest % Test
+  )
+
+  val client = libraryDependencies ++= Seq(
+    "com.typesafe.play" %% "play-ws" % PlayVersion,
+    "io.dropwizard.metrics" % "metrics-core" % "3.1.2"
+  )
+
+  val `client-javadsl` = libraryDependencies ++= Nil
+
+  val `client-scaladsl` = libraryDependencies ++= Seq(
+    scalaTest % Test
+  )
+
+  val `integration-client-javadsl` = libraryDependencies ++= Nil
+
+  val server = libraryDependencies ++= Seq(
+    // todo probably not needed when #530 is done
+    "com.typesafe.akka" %% "akka-actor" % AkkaVersion
+  )
+
+  val `server-javadsl` = libraryDependencies ++= Seq(
+    // todo probably not needed when #530 is done
+    "com.typesafe.akka" %% "akka-actor" % AkkaVersion,
+    guava
+  )
+
+  val `server-scaladsl` = libraryDependencies ++= Seq(
+    "com.typesafe.akka" %% "akka-actor" % AkkaVersion,
+    scalaTest % Test
+  )
+
+  val `testkit-core` = libraryDependencies ++= Seq(
+    "com.typesafe.akka" %% "akka-actor" % AkkaVersion,
+    "com.typesafe.akka" %% "akka-stream" % AkkaVersion
+  )
+
+  val `testkit-javadsl` = libraryDependencies ++= Seq(
+    "com.typesafe.play" %% "play-netty-server" % PlayVersion,
+    "org.apache.cassandra" % "cassandra-all" % CassandraAllVersion exclude("io.netty", "netty-all"),
+    "com.typesafe.akka" %% "akka-stream" % AkkaVersion,
+    "com.typesafe.akka" %% "akka-stream-testkit" % AkkaVersion,
+    "com.typesafe.akka" %% "akka-persistence-cassandra" % AkkaPersistenceCassandraVersion,
+    scalaTest % Test,
+    scalaJava8Compat
+  )
+
+  val `testkit-scaladsl` = libraryDependencies ++= Seq(
+    "com.typesafe.play" %% "play-netty-server" % PlayVersion,
+    "org.apache.cassandra" % "cassandra-all" % CassandraAllVersion exclude("io.netty", "netty-all"),
+    "com.typesafe.akka" %% "akka-stream-testkit" % AkkaVersion,
+    "com.typesafe.akka" %% "akka-persistence-cassandra" % AkkaPersistenceCassandraVersion,
+    scalaTest % Test
+  )
+
+  val `integration-tests-javadsl` = libraryDependencies ++= Seq(
+    "com.typesafe.play" %% "play-netty-server" % PlayVersion,
+    "com.novocode" % "junit-interface" % "0.11" % "test",
+    scalaTest
+  )
+
+  val `integration-tests-scaladsl` = libraryDependencies ++= Seq(
+    "com.typesafe.play" %% "play-netty-server" % PlayVersion,
+    "com.novocode" % "junit-interface" % "0.11" % "test",
+    scalaTest
+  )
```

Comment:
When writing tests I like having options, so even thought these are `integration-tests-{scala,java}dsl`, I like that both include `scalaTest` and `junit`.